leader-followers/src/main/java/com/iluwatar/leaderfollowers/App.java

```diff
@@ -1,61 +1,16 @@
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
 package com.iluwatar.leaderfollowers;
 
 import java.security.SecureRandom;
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-/**
- * Leader/Followers pattern is a concurrency pattern. This pattern behaves like a taxi stand where
- * one of the threads acts as leader thread which listens for event from event sources,
- * de-multiplexes, dispatches and handles the event. It promotes the follower to be the new leader.
- * When processing completes the thread joins the followers queue, if there are no followers then it
- * becomes the leader and cycle repeats again.
- *
- * <p>In this example, one of the workers becomes Leader and listens on the {@link TaskSet} for
- * work. {@link TaskSet} basically acts as the source of input events for the {@link Worker}, who
- * are spawned and controlled by the {@link WorkCenter} . When {@link Task} arrives then the leader
- * takes the work and calls the {@link TaskHandler}. It also calls the {@link WorkCenter} to
- * promotes one of the followers to be the new leader, who can then process the next work and so on.
- *
- * <p>The pros for this pattern are: It enhances CPU cache affinity and eliminates unbound
- * allocation and data buffer sharing between threads by reading the request into buffer space
- * allocated on the stack of the leader or by using the Thread-Specific Storage pattern [22] to
- * allocate memory. It minimizes locking overhead by not exchanging data between threads, thereby
- * reducing thread synchronization. In bound handle/thread associations, the leader thread
- * dispatches the event based on the I/O handle. It can minimize priority inversion because no extra
- * queuing is introduced in the server. It does not require a context switch to handle each event,
- * reducing the event dispatching latency. Note that promoting a follower thread to fulfill the
- * leader role requires a context switch. Programming simplicity: The Leader/Followers pattern
- * simplifies the programming of concurrency models where multiple threads can receive requests,
- * process responses, and de-multiplex connections using a shared handle set.
- */
+@Slf4j
 public class App {
+  private static final Logger log = LoggerFactory.getLogger(App.class);
```

Comment:
No need to declare the logger like this. After inserting the annotation you can just log like `LOGGER.info("hello")`